pubspec.yaml

```diff
@@ -18,16 +18,10 @@ dependencies:
 
```

Comment:
All of the comments above ⬆️  can be removed now, right?

Reply:
Nice catch, thank you!
Fixed.